flake.nix

```diff
@@ -1,51 +1,263 @@
-# NOTE: This flake is only used for source pinning and has no outputs
```

Comment:
flake.lock has been renamed to old.flake.lock
nx has been renamed to old.flake.nix 

the new flake+lock have been moved from __std__ to the top-level.
But git doesn't see that as a "move", but as a diff. 
So this is a diff between the current and new flake. It's a mess, it can be skipped over

Reply:
The reason this was done is: 
We need a way to test the new flake-based CI, so we need the new flake to be at the top-level.
Fortunately, the old flake was only used for source pinning, so it's ok to rename it and leave it as `old.flake.nix`.

Reply:
Unless...
What if we changed this line https://github.com/input-output-hk/ci-ops/blob/fcbd80ad860405ad947bc95eb707db10f081de53/jobsets/default.nix#L374
from `github:input-output-hk/plutus` to `github:input-output-hk/plutus/__std__` 
Then it should pick up the flake in `github:input-output-hk/plutus/__std__/flake.nix`
And so we can keep the original flake.nix as is in the top-level
What do you think @jbgi?

Reply:
No, think hydra will look at the `__std__` branch instead of subdirectory.
I think the way to go is to add the sub-flake as input to the main flake (with `url = path:./__std__`) and expose the `hydraJobs` attribute in the main flake.

Reply:
> I think the way to go is to add the sub-flake as input to the main flake (with url = path:./__std__) and expose the hydraJobs attribute in the main flake.

If that works it would be nice!